pages/mesosphere/dcos/services/elastic/2.7.0-6.8.1/release-notes/index.md

```diff
@@ -0,0 +1,89 @@
+    -   `elasticsearch.plugin_https_proxy_port`
+
+    Changes: ([#4](https://github.com/mesosphere/dcos-elastic-service/pull/4))
+-   Added custom domain support for Elastic. New setting is `service.security.custom_domain`. This custom domain is used in place of the default `autoip.dcos.thisdcos.directory`. This can be used to expose the service securely outside of the cluster, but requires setting up external DNS. Check [this documentation page](https://mesosphere.github.io/dcos-commons/developer-guide/#externalizing-transport-encryption-and-security) for more information. ([#9](https://github.com/mesosphere/dcos-elastic-service/pull/9))
+-   Made it possible to scale down the amount of coordinator nodes. ([#15](https://github.com/mesosphere/dcos-elastic-service/pull/15))
+-   Added custom YAML configuration support to Kibana. New setting is: `kibana.custom_kibana_yml` ([#14](https://github.com/mesosphere/dcos-elastic-service/pull/14))
+-   Added plugin installation support for Kibana. New setting is: `kibana.plugins` ([#16](https://github.com/mesosphere/dcos-elastic-service/pull/16))
+-   Added volume profile support. New settings are:
+
+    -   `master_nodes.volume_profile`
+    -   `data_nodes.volume_profile`
+    -   `ingest_nodes.volume_profile`
+    -   `coordinator_nodes.volume_profile`
+
+    Check the [Storage Service documentation](https://docs.d2iq.com/mesosphere/dcos/services/storage/1.0.0/) for more details. ([#18](https://github.com/mesosphere/dcos-elastic-service/pull/18))
+-   Added placement constraints capability to Kibana. New setting is `kibana.placement` (array of placement constraint arrays). Check the [Marathon placement constraints documentation](https://mesosphere.github.io/marathon/docs/constraints.html) for more details. ([#27](https://github.com/mesosphere/dcos-elastic-service/pull/27))
+-   Added virtual network (CNI) support to Kibana. New settings are:
+
+    -   `virtual_network_enabled` (boolean)
+    -   `virtual_network_name` (string, default `dcos`)
+    -   `virtual_network_plugin_labels` (array of objects with `key` and `value` keys)
+
+    Changes: ([#28](https://github.com/mesosphere/dcos-elastic-service/pull/28))
+-   Added scheduler-plan based health checks. ([#3069](https://github.com/mesosphere/dcos-commons/pull/3069))
+
+
+<a id="orgae15112"></a>
+
+## Bug Fixes
+
+-   Fixed Kibana service URL (`<cluster-url>/service/kibana`). This makes Kibana's service URL work out of the box even when X-Pack Security is enabled. This was a limitation in the previous version, where we required Kibana to be exposed with EdgeLB. Which is still possible, but not required anymore. ([#3036](https://github.com/mesosphere/dcos-commons/pull/3036))
+-   Fixed issue where setting `http.cors.allow-origin` to `*` would cause the deploy to fail to due the Elasticsearch configuration file be compiled with incorrectly escaped content. ([#23](https://github.com/mesosphere/dcos-elastic-service/pull/23))
```

Comment:
```suggestion
-   Fixed issue where setting `http.cors.allow-origin` to `*` would cause the deployment to fail due to the Elasticsearch configuration file being compiled with incorrectly escaped content. ([#23](https://github.com/mesosphere/dcos-elastic-service/pull/23))
```